### DISCUSSION TITLE
Account restricted for violation of terms of service?

### DISCUSSION THREAD

**Discussion by w0rxbend** [codewars/codewars.com#3032] [Q&A] [answered]
Hello. Explain to me, please, why [my account](https://www.codewars.com/users/limpid-kzonix) is blocked. I have this issue with the previous account, later I deleted it and created a new one. passed a couple of katas and now I get this again?

**Comment by Blind4Basics** [comment, marked as answer]
Hello,

This account isn't locked.
Two previous accounts were, yes. First, for obvious reasons. Second, I'm not entirely sure why/how... But well, you can use your new account, ofc. Just don't copy solutions from the internet again...

Cheers
